multivac/fetch.py

```diff
@@ -213,6 +215,13 @@ def download_log(self):
         r = http_get(url)
         self.log = r.content
 
+    def check_pat_limit(self):
```

Comment:
This method does not really check the limit, because a check should lead to action. If limit is okay, continue downloading, if it's low, do this other thing.

Reply:
I just looked at the headers of https://api.github.com/users/lastochka42 and https://api.github.com/users/NickVolynkin. The maximum value of `x-ratelimit` is 60. It will be `% 100 == 0` when it's zero.

<img width="174" alt="Screenshot 2022-06-26 at 11 39 06" src="https://user-images.githubusercontent.com/8015154/175799636-38fb854b-774f-4f99-8fa6-b5c9488dd466.png">